gatsby-node.js

```diff
@@ -13,16 +13,30 @@ exports.createPages = ({ graphql, actions }) => {
             node {
               slug
             }
+            next {
+              slug
+            }
+            previous {
+              slug
+            }
           }
         }
       }
     `).then(result => {
-      result.data.allDatoCmsObra.edges.map(({ node: obra }) => {
+      result.data.allDatoCmsObra.edges.map(({node, next, previous}) => {
```

Comment:
@guayom gracias! mae, que deberiamos de leer para profundizar en esto? gatsby lifecycle?

Reply:
Esto es lo que provee Dato. Al hacer un query de allDatoCmsObra podemos ver que nos regresa node, next y previous. 

Ahora, la sintaxis de que usé se llama object destructuring. En el arrow function, en vez de pasar simplemente `node`, necesito pasar `next` y `preview` porque vienen en el objeto que devuelve el query.

Destructuring:
https://developer.mozilla.org/es/docs/Web/JavaScript/Referencia/Operadores/Destructuring_assignment

Reply:
me referia a la parte de pasar por contexto la data, pero, me contestaste por slack 👍 

@JgeovanniAm @glorianaa80 @NycollSM son estos links, igual se los pase por slack

- https://www.gatsbyjs.org/docs/gatsby-lifecycle-apis/
- https://www.gatsbyjs.org/docs/overview-of-the-gatsby-build-process/